qadence/noise/readout.py

```diff
@@ -120,7 +120,7 @@ def create_noise_matrix(
 
 
 def bs_corruption(
-    err_idx: list,
+    err_idx: torch.Tensor,
```

Comment:
```suggestion
    from torch import Tensor
    err_idx: Tensor,
```

It is more idiomatic to Qadence style.